doc/contributors-guide/making-and-submitting-changes.md

```diff
@@ -2,11 +2,15 @@
 
 Add Git workflow intro here
 
-### Add your changes using the GitHub UI&#x20;
+### Documentation repo:Add your changes using the GitHub UI&#x20;
 
 Add video here
 
-Add text content
+* Fork the documentation repo (Additional information: The default branch of the documentation repo has been set  to develop branch. So on the "Create a new fork" page, you can leave the "Copy the develop branch only" checkbox as checked since you will need the develop branch for the next steps).
+* Confirm that you are in the develop branch of your fork.
+* Create a new branch from the develop branch and make your changes in the new branch you created.
+* Send a pull request pointing to the develop branch of the original documentation repo.
+* Wait for review of your changes from the repo admin(s).
```

Comment:
Some things here in the list were links and you did not format them as links that the reader can click on. Check the formattings... Links and any other type of formatting, and add them.

Reply:
I have corrected and made changes @Ifycode